### DISCUSSION TITLE
Sink to Hub replication

### DISCUSSION THREAD

**Discussion by xinix00** [ravendb/ravendb#12713] [Q&A] [answered]
Hi,

I'm trying to create a Hub <-> Sink replication, however, it is only replicating from the Hub to the Sink, not the other way around.

The Hub is unable to access the Sink (I thought this was the design), the Sink however can reach the Hub. The hub is replicated to the Sink, however when I edit a document on the Sink, the document is not transferred to the Hub.

Is there something I'm doing wrong? Of course "Allow replication from Sink to Hub" is enabled on both ends.

**Comment by haludi** [comment]
Hi,

Do you use secured servers with a certificate?

Best regards,
Igal

**Comment by xinix00** [comment, marked as answer]
No, currently inside a docker, unsecured.

**Reply by ayende** [reply]
Pull replication will work without security, but to push data you need to ensure that you are running in a secured mode.
